src/main/java/io/split/android/client/cache/SplitChangeCache.java

```diff
@@ -18,28 +17,27 @@
 
 public class SplitChangeCache implements ISplitChangeCache {
 
-    private ISplitCache _splitCache;
+    private ISplitCache mSplitCache;
 
     public SplitChangeCache(IStorage storage) {
-        this._splitCache = new SplitCache(storage);
+        this.mSplitCache = new SplitCache(storage);
     }
 
     @Override
     public boolean addChange(SplitChange splitChange) {
-        if (_splitCache == null) return false;
+        if (mSplitCache == null) return false;
         boolean result = true;
-        _splitCache.setChangeNumber(splitChange.till);
-        for (Split split :
-                splitChange.splits) {
-            result = result && _splitCache.addSplit(split.name, Json.toJson(split));
+        mSplitCache.setChangeNumber(splitChange.till);
```

Comment:
can till be null?

Reply:
till is a primitive type (long), so it can't be null.